tests/test-team/template-mgmt-component-tests/nhs-app/template-mgmt-preview-nhs-app-page.component.spec.ts

```diff
@@ -80,6 +99,35 @@ test.describe('Preview NHS App template Page', () => {
     );
   });
 
+  test.describe('Routing feature flag enabled', () => {
+    test.use({ storageState: { cookies: [], origins: [] } });
+
+    test('when user visits page, then page is loaded, with routing Enabled', async ({
```

Comment:
same as below, already stated in the describe
```suggestion
    test('when user visits page, then page is loaded', async ({
```

Reply:
Updated. 

See: https://github.com/NHSDigital/nhs-notify-web-template-management/pull/691#discussion_r2362416050